runtime/jcl/common/sigusr2.c

```diff
@@ -0,0 +1,133 @@
+/*******************************************************************************
+ * Copyright (c) 2022, 2022 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+#include "j9user.h"
+#include "j9.h"
+#include "j9protos.h"
+#include "j9consts.h"
+#include "omrthread.h"
+#include "j9dump.h"
+#include "jni.h"
+#include "vmaccess.h"
+#include "jvminit.h"
+
+#include "jcl_internal.h"
+#include "j9port.h"
+#include "ut_j9jcl.h"
+
+#define J9_SIGUSR2_CYCLE_MSEC  200
+
+static UDATA sigUsr2Handler(struct J9PortLibrary* portLibrary, void* userData);
+static UDATA sigUsr2Wrapper(struct J9PortLibrary* portLibrary, U_32 gpType, void* gpInfo, void* userData);
```

Comment:
nit: `*` binds more tightly to the parameter name, so the space should precede rather than follow `*`:
```
static UDATA sigUsr2Handler(struct J9PortLibrary *portLibrary, void *userData);
static UDATA sigUsr2Wrapper(struct J9PortLibrary *portLibrary, U_32 gpType, void *gpInfo, void *userData);
```
Please format likewise throughout.

Reply:
So it should be, for example, `struct J9PortLibrary* portLibrary` instead?

Reply:
Nevermind, somehow my brain interpreted your correction as the original diff.